src/afl-showmap.c

```diff
@@ -986,7 +985,7 @@ int main(int argc, char **argv_orig, char **envp) {
   if (getenv("AFL_DEBUG")) {
 
     DEBUGF("");
-    for (i = 0; i < argc; i++)
+    for (int i = 0; i < argc; i++)
```

Comment:
some compilers need a special command line option to support this and fail otherwise (I think this is C99)

Reply:
Fair point - that does require C99 or a compiler with extensions.
Then again, so does `//` comments.
Is there a minimum target compiler documentation somewhere I can review?

Reply:
In the meantime, we can undo the changes to 741 and 989?